File: web/src/api/mall/member.js
// 商城会员管理 API（后台）
import service from '@/utils/request'

export const getMemberList = (data) => {
  return service({
    url: '/mall/member/list',
    method: 'post',
    data
  })
}

export const getMemberDetail = (data) => {
  return service({
    url: '/mall/member/detail',
    method: 'post',
    data
  })
}

File: server/router/admin/mall/member.go
// member.go 商城后台会员路由注册。
package mall

import "github.com/gin-gonic/gin"

// MemberRouter 后台会员管理路由。
type MemberRouter struct{}

// InitMemberRouter 注册 /mall/member/* 路由（需后台 JWT + Casbin）。
func (r *MemberRouter) InitMemberRouter(Router *gin.RouterGroup) {
	member := Router.Group("mall").Group("member")
	{
		member.POST("list", memberApi.GetMemberList)
		member.POST("detail", memberApi.GetMemberDetail)
	}
}
